src/components/Settings.vue

```diff
@@ -1043,7 +1043,7 @@
 										cols="6"
 									>
 										<v-switch
-											hint="Disable this to prevent applications like Home Assistant to automatically detect and connect to your zwavejs2mqtt instance"
+											hint="Disable this to prevent applications like Home Assistant to automatically detect and connect to your zwave-js-ui instance"
```

Comment:
```suggestion
											hint="Disable this to prevent applications like Home Assistant to automatically detect and connect to your Z-Wave JS UI instance"
```